e2e/dr_test.go

```diff
@@ -21,8 +22,8 @@ func TestDR(dt *testing.T) {
 		t.Fatal("No tests found in the configuration file")
 	}
 
-	if err := util.ValidateClustersInDRPolicy(Ctx.env, Ctx.config, Ctx.log); err != nil {
-		t.Fatalf("Failed to validate clusters with DRPolicy: %s", err)
+	if err := validate.TestConfig(Ctx.env, Ctx.config, Ctx.log); err != nil {
+		t.Fatalf("Failed to validate test environment with DR configurations: %s", err)
```

Comment:
This is not a test environment in all cases. Lets return a good error in validate.TestConfig() and just use t.Fatal(err) here.